src/replicache.ts

```diff
@@ -163,10 +168,29 @@ export default class Replicache implements ReadTransaction {
     return res.root;
   }
 
+  private _setRoot(root: Promise<string | undefined>) {
+    this._root = root;
+    this._setStorage(root);
+  }
+
+  private async _setStorage(root: Promise<string | undefined>) {
+    // Also set an item in localStorage so that we can synchronize multiple
+    // windows/tabs.
+    localStorage[storageKeyName(this._name)] = await root;
```

Comment:
Is it possible to use `sessionStorage`? Seems like it might be faster.

Reply:
I though `sessionStorage` would have been enough but it is not shared between tabs.

Reply:
blech